gatsby/content/projects/2016-02-10-matrix-appservice-gitter.mdx

```diff
@@ -14,6 +14,8 @@ bridges: Gitter
 featured: true
 ---
 
+**Note:** This project is now deprecated in favor of the native Gitter bridge
+
 This project bridges [Gitter](https://gitter.im) to Matrix, via the AS API on the Matrix side, and a Gitter user on the Gitter side.
 
 Get it from [github](https://github.com/matrix-org/matrix-appservice-gitter).
```

Comment:
What is the difference between the gatsby and jekyll stuff?

I see this information is also at [`jekyll/_posts/projects/2018-11-20-bridges.md#gitter`](https://github.com/matrix-org/matrix.org/blob/a05cdebec41c24214cdacfdce0b171abaf451195/jekyll/_posts/projects/2018-11-20-bridges.md#gitter) but I am not sure what it corresponds to.

Reply:
The jekyll content is not used in the build, we should remove it but there might be some legacy value...